api/src/service/domain/organization/user_create.ts

```diff
@@ -66,8 +66,13 @@ export async function createUser(
     additionalData: data.additionalData || {},
   };
 
+  const unfinishedBusinessEvent = UserCreated.createEvent(source, publisher, eventTemplate);
```

Comment:
**unfinishedBusinessEvent** could here also be a valid create Event.
I would suggest to rename it to createEvent and delete line 98 to prevent executing createEvent two times.